### PR TITLE
fix(loadgen): remove bound usages of E proxy methods

### PR DESCRIPTION
**Pull request by mhofman**
No reason to rely on this, as there are arguments to make this "feature" go away in the future: https://github.com/endojs/endo/issues/1253

### CODE REVIEW

File: loadgen/contract/agent-prepare-loadgen.js [hunk 1]
```diff
@@ -300,7 +300,9 @@ export default async function startAgent({
   // Use `when` as older versions of agoric-sdk cannot accept a promise
   // See https://github.com/Agoric/agoric-sdk/issues/3837
   /** @type {ERef<import('../types.js').AttenuatedAMM>} */
-  const amm = E.when(withFee(ammInstance), E(zoe).getPublicFacet);
+  const amm = E.when(withFee(ammInstance), (instance) =>
```
Comment (erights) [inline loadgen/contract/agent-prepare-loadgen.js:303]: The parens around the single-identifier-parameter surprises me. Are we not linting this repository?

**Review by erights — APPROVED** [review]
LGTM, thanks!

File: loadgen/contract/agent-prepare-loadgen.js (hunk 1 above)
Comment (erights) [reply loadgen/contract/agent-prepare-loadgen.js:303]:
```suggestion
  const amm = E.when(withFee(ammInstance), instance =>
```

Comment (mhofman) [reply loadgen/contract/agent-prepare-loadgen.js:303]: The prettier rules in this repo are different (I updated to prettier 2 much before agoric-sdk did, and kept with the defaults)

Comment (erights) [reply loadgen/contract/agent-prepare-loadgen.js:303]: Please file a bug to fix the prettier rules on this repository (and others?) to remove these parens. Thanks.

Comment (mhofman) [reply loadgen/contract/agent-prepare-loadgen.js:303]: Filed: https://github.com/Agoric/testnet-load-generator/issues/94

**Pull request merged**